cmd/proxy/actions/home.go

```diff
@@ -1,10 +1,9 @@
 package actions
 
 import (
-	"github.com/gobuffalo/buffalo"
 	"net/http"
 )
 
-func proxyHomeHandler(c buffalo.Context) error {
-	return c.Render(http.StatusOK, proxy.JSON("Welcome to The Athens Proxy"))
+func proxyHomeHandler(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte(`"Welcome to The Athens Proxy"`))
```

Comment:
We can have one default helper function to make sure that everything is always a json response.

Reply:
Yes, but let's do it in a future pr